v2/examples/spiffe-http/README.md

```diff
@@ -0,0 +1,114 @@
+# gRPC SVID example 
+
+This example shows how two services using http can communicate using mTLS with X509 SVIDs obtained from SPIFFE workload API.
+
+Each service is connecting to workload API to fetch its identities. Since this example assumes the SPIRE implementation, it uses the SPIRE default socket path: `/tmp/agent.sock`. 
```

Comment:
```suggestion
Each service is connecting to the Workload API to fetch its identities. Since this example assumes the SPIRE implementation, it uses the SPIRE default socket path: `/tmp/agent.sock`. 
```